Code/notebooks/bootcamp_pandas-shape.ipynb

```diff
@@ -1,1296 +1,1356 @@
 {
+ "nbformat_minor": 0, 
+ "nbformat": 4, 
  "cells": [
   {
-   "cell_type": "markdown",
+   "source": [
+    "# Pandas 3:  Shaping data  \n", 
+    "\n", 
+    "The second in a series of notebooks that describe Pandas' powerful data management tools.  This one covers shaping methods:  switching rows and columns, pivoting, and stacking.  We'll see that this is all about the indexes:  the row and column labels.  \n", 
+    "\n", 
+    "Outline:  \n", 
+    "\n", 
+    "* [Example:  WEO debt and deficits](#wants).  Something to work with.  \n", 
+    "* [Indexing](#index).  Setting and resetting the index.  Multi-indexes. \n", 
+    "* [Switching rows and columns](#pivot).  Transpose.  Referring to variables with multi-indexes.  \n", 
+    "* [Stack and unstack](#stack).  Managing column structure and labels.  \n", 
+    "* [Pivot](#pivot).  Unstack shortcut if we start with wide data. \n", 
+    "* [Review](#review).  Apply what we've learned.  \n", 
+    "\n", 
+    "More data management topics coming.  \n", 
+    "\n", 
+    "**Note: requires internet access to run.**  \n", 
+    "\n", 
+    "<!-- \n", 
+    "internal links http://sebastianraschka.com/Articles/2014_ipython_internal_links.html\n", 
+    "-->\n", 
+    "\n", 
+    "This Jupyter notebook was created by Dave Backus, Chase Coleman, and Spencer Lyon for the NYU Stern course [Data Bootcamp](http://databootcamp.nyuecon.com/).  \n", 
+    "\n", 
+    "## tl;dr\n", 
```

Comment:
I added this tl;dr section as a brief summary of the main tools from the chapter.

I'm not 100% sure I like it, what do you think?